### ISSUE TITLE
pytest --testmon fails with INTERNALERROR

### ISSUE THREAD

**Issue by brendanator** [tarpas/pytest-testmon#175]
When I run testmon with a test that runs [cookiecutter](https://github.com/cookiecutter/cookiecutter) using the `cookies` fixture like so:

```
def test_default_settings(cookies, scripts_dir):
    result = cookies.bake(
        template=str(Path(scripts_dir, "cookiecutter-template"))
    )

    assert ...
``` 

testmon fails horribly. I suspect it's because it ends up trying to parse a cookiecutter template file
```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/_pytest/main.py", line 268, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/_pytest/main.py", line 322, in _main
INTERNALERROR>     config.hook.pytest_runtestloop(session=session)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
INTERNALERROR>     return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/_pytest/main.py", line 347, in pytest_runtestloop
INTERNALERROR>     item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
INTERNALERROR>     return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/pluggy/_callers.py", line 55, in _multicall
INTERNALERROR>     gen.send(outcome)
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/testmon/pytest_testmon.py", line 254, in pytest_runtest_protocol
INTERNALERROR>     self.testmon.stop_and_save(
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/testmon/testmon_core.py", line 372, in stop_and_save
INTERNALERROR>     self.stop()
INTERNALERROR>   File "/Users/brendan/.local/share/virtualenvs/core-bRkH1-KB/lib/python3.9/site-packages/testmon/testmon_core.py", line 194, in node_data_from_cov
INTERNALERROR>     module.lines, module.special_blocks, covered
INTERNALERROR> AttributeError: 'Module' object has no attribute 'special_blocks'
```

**Comment by tarpas** [comment]
That's quite strange. Anyway, 
pytest-testmon==1.3.0rc3 has quite different code around here. Could you please test that version?

**Comment by tarpas** [comment]
This has been fixed with v 1.3.0. Please reopen if you encounter the bug.
